packages/ai-jsx/src/batteries/sidekick/platform/system-message.tsx

```diff
@@ -69,10 +80,11 @@ export function SidekickSystemMessage({
   }
 
   return (
-    <>
+    <SystemMessage>
       {dateTimeSystemMessage}
-      {responseFramingSystemMessage}
-      {outputFormatToUse === 'text/markdown' && <SystemMessage>Respond with Markdown.</SystemMessage>}
+      {hasKnowledgeBase && responseFormatSystemMessage}
+      {styleSystemMessage}
```

Comment:
These will be joined without whitespace, e.g. "...just answer as if you know the answer.Speak politely but casually.Respond with Markdown."

It's perhaps unlikely to matter, but I guess I could imagine the model responding like that if enough of the prompt is like that?

Reply:
Aah, I was wondering what would happen and instead of trying it I asked Perplexity and seems like the answer was wrong. Why is the formatting so weird? Just apply the conditionals first, then do HTML formatting :(

But yeah, this should not matter really. The result looks weird which I'll probably fix, but it seems like LLMs are comfortable even if a lot of the small details are omitted, not just spaces but stopwords and all.

Reply:
Done.